### PR TITLE
[Do not merge] Link docker repositories

### PR DESCRIPTION
**Pull request by mnagy**
Work in progress, should be post 3.0.

### CODE REVIEW

**Comment by smarterclayton** [comment]
Is this going to be rebased?

**Comment by mnagy** [comment]
Am busy ATM with other stuff. Will try to get back to this next week, unless you want it faster.

File: assets/app/scripts/filters/util.js [hunk 1]
```diff
@@ -162,6 +162,15 @@ angular.module('openshiftConsole')
       return link;
     };
   })
+  .filter('dockerHubRepository', function() {
+    return function(repository) {
+      // Check if repository is in the form organization/repository
+      if (repository.match(/[a-zA-Z0-9-_.]+\/[a-zA-Z0-9-_.]/)) {
```
Comment (spadgett) [inline assets/app/scripts/filters/util.js]: I know this is a work in progress, but I noticed this regex will match garbage at the start and end of the string.

openshift_web_console

Nit: `String.search() !== -1` should be faster here than `String.match()`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/search

**Comment by mnagy** [comment]
Closing, will be finished by the UI team.

**Pull request closed without merging**